matplotlib/napari-matplotlib
# Misc fixes needed to build docs locally

Does what it says in the title.

To test locally:
```bash
python -m pip install -e ".[docs]"
cd docs
make html
```

The HTML docs should then be built succesfully in the `_build/html` directory.
